sdk/src/main/java/io/radar/sdk/model/RadarEvent.kt

```diff
@@ -319,6 +340,7 @@ class RadarEvent(
         obj.putOpt(FIELD_ACTUAL_CREATED_AT, RadarUtils.dateToISOString(this.actualCreatedAt))
         obj.putOpt(FIELD_LIVE, this.live)
         obj.putOpt(FIELD_TYPE, stringForType(this.type))
+        obj.putOpt(FIELD_CUSTOM_TYPE, this.customType)
```

Comment:
do we need `FIELD_METADATA` to be serialized to json as well?

Reply:
It's fixed now.